### DISCUSSION TITLE
36B69FAC-D4D5-47AC-B52D-D515F2E573B3

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#172] [General]
# 36B69FAC-D4D5-47AC-B52D-D515F2E573B3

This addon adds Saturn's polar auroras.

https://celestia.mobi/addon?item=36B69FAC-D4D5-47AC-B52D-D515F2E573B3

**Comment by xXLouisPlayzXx** [comment]
Bro this looks fire 🔥🔥🔥

**Reply by SiriusBislifeless** [reply]
🔥🔥🔥🔥🔥
